src/components/ArrayCard.tsx

```diff
@@ -4,16 +4,12 @@ import { isEmpty } from 'lodash';
 
 import * as Antd from 'antd';
 
-import { ICommonCardProps } from '../interfaces';
 import { IModel } from '../props';
 
-import Card from './Card';
+import Card, { ICardProps } from './Card';
 
-export interface IArrayCardProps extends ICommonCardProps {
-  children?: any;
-  isLoading?: boolean;
-  model: IModel;
-  renderTopRight?: () => any;
+export interface IArrayCardProps extends ICardProps {
+    model: IModel[];
```

Comment:
Well `IModel` is VERY vague. It's eventually going to be vaguely object-like probably but right now it's just `any`. This is more like saying 'the one thing we know about the model for THIS component is that you should be able to iterate over it'. I think model still makes sense since it will still look like `model={case.liens}` which is like... the thing that is being rendered? IDK.

Reply:
i see both arguments. the model prop is common among most components, so it's a little strange to name this something else. On the other hand, it really is a `modelList` or something... I think naming this model, or renaming the model prop makes the most sense in this scenario so that there's a uniform API

Reply:
That being said, this value should probably default to empty array